lua/lvim/core/telescope.lua

```diff
@@ -71,18 +13,20 @@ function M.config()
   if not ok then
     return
   end
-  lvim.builtin.telescope = vim.tbl_extend("force", lvim.builtin.telescope, {
+  lvim.builtin.telescope = {
+    active = true,
+    on_config_done = nil,
     defaults = {
       prompt_prefix = lvim.icons.ui.Telescope .. " ",
       selection_caret = lvim.icons.ui.Forward .. " ",
       entry_prefix = "  ",
       initial_mode = "insert",
       selection_strategy = "reset",
       sorting_strategy = "descending",
-      layout_strategy = "horizontal",
+      theme = "dropdown",
```

Comment:
I don't think it's safe to pass this one here, let's move it outside of the `default` table to avoid confusion.

Reply:
the theme sets it so I think it's ok https://github.com/nvim-telescope/telescope.nvim/blob/cabf991b1d3996fa6f3232327fc649bbdf676496/lua/telescope/themes.lua#L30

Reply:
but you're right it may be better to move it out since it's not a telescope config option

Reply:
although we'd also need to lvim.builtin.telescope.defaults.theme = nil after loading the theme in that case